dts/msc-sm2-imx6x.dtsi

```diff
@@ -12,7 +12,7 @@
 #include "skeleton.dtsi"
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/sound/fsl-imx-audmux.h>
-
+#include <dt-bindings/regulator/dlg,da9063-regulator.h>
```

Comment:
Just checking - not a typo?

Reply:
It is not a typo, it is the real (weird) name of the file, he he. Someone decided to use a comma instead of a regular dash between the vendor name and the chip name!